accounts/serializers.py

```diff
@@ -1,6 +1,6 @@
-from rest_framework import serializers
+from django.db.models import fields
```

Comment:
why is this required?

Reply:
Yes, this import was not required. Removed it in the recent push.